net-libs/libwebsockets/metadata.xml

```diff
@@ -19,12 +19,15 @@
 		libwebsockets is a C library designed to make development of websockets clients and servers easier (possible).
 	</longdescription>
 	<use>
-		<flag name="ssl">Include SSL support (default <pkg>dev-libs/openssl</pkg>)</flag>
-		<flag name="zlib">Include zlib support</flag>
-		<flag name="static-libs">Build the static version of the library</flag>
+		<flag name="client">The client part of the library</flag>
 		<flag name="http2">Compile with support for http2</flag>
 		<flag name="ipv6">Compile with support for ipv6</flag>
+		<flag name="libev">Compile with support for libev</flag>
+		<flag name="libressl">Include <pkg>dev-libs/libressl</pkg> support instead of <pkg>dev-libs/openssl</pkg>. Available iff ssl is enabled</flag>
+		<flag name="libuv">Compile with support for libuv</flag>
 		<flag name="server">The server part of the library</flag>
-		<flag name="client">The client part of the library</flag>
+		<flag name="ssl">Include SSL support (default <pkg>dev-libs/openssl</pkg>)</flag>
```

Comment:
$ quse -D ssl
 global:ssl: Add support for Secure Socket Layer connections
local:ssl:net-libs/libwebsockets: Include SSL support (default dev-libs/openssl)

Similar with libressl. It is up to you whether they should be kept or removed. Looks redundant to me.